### PR TITLE
add new tech to the wiki

### PR DESCRIPTION
**Pull request by ghost-hikaru**

### CODE REVIEW

File: temp/wiki.md [hunk 1]
```diff
@@ -11,34 +11,59 @@
 - API Star
 - API SCNF
 - API navita
-- Collecteur 
-- Module DB
-- Unité central de controle
+- Collecteur  
+- Unité central de controle et de stockage
```
Comment (Slonev0) [inline temp/wiki.md]: Centrale* et contrôle*

**Review by Slonev0 — CHANGES_REQUESTED** [review]
just need to correct a spelling mistake after

**Review by Payll — APPROVED** [review]
Remove .DS_Store after

**Review by Slonev0 — APPROVED** [review]
thx :)

**Pull request merged**